test/WebJobs.Script.Tests/TestMetricsLogger.cs

```diff
@@ -55,7 +55,7 @@ public void EndEvent(MetricEvent metricEvent)
 
         public void LogEvent(string eventName, string functionName = null)
         {
-            LoggedEvents.Add(MetricsEventManager.GetAggregateKey(eventName, functionName));
+            LoggedEvents.Add(eventName.ToLowerInvariant());
```

Comment:
I think this is actually correct as is - are you saying tests are failing without your change?

Reply:
Fixed test to look for key that include _functionName